queue/abstract.lua

```diff
@@ -11,6 +11,10 @@ local qc       = require('queue.compat')
 local num_type = qc.num_type
 local str_type = qc.str_type
 
+-- since:
+-- https://github.com/tarantool/tarantool/commit/f266559b167b4643c377724eebde9651877d6cc9
+local is_ddl_txn_supported = qc.check_version({2, 2, 1})
```

Comment:
I suggest not using the prefix `is_` as we do not use it in boolean `temporary`, `unique`...

Reply:
Thank you, as you wish.